ccmlib/scylla_node.py

```diff
@@ -39,6 +39,75 @@
 if TYPE_CHECKING:
     from ccmlib.scylla_cluster import ScyllaCluster
 
+class RepairOptions:
```

Comment:
I am not sold on these classes, they are internal only and are used only once in one method. Do we really need them? Cannot we design the method better, or maybe split it in two so we dont need them at all?

Reply:
Another idea would be to have `run_repair` and `run_cluster_repair` use these classes as well, which would simplify the `repair` method as well